### PR TITLE
#1021 redesign the floating windows - functional groups window

### PR DESCRIPTION
**Pull request by AnaMoskvina**
Change grid layout and background color of items in TemplateLable, update EmptySearchResult view

Current result:
Functional_group
Template_library
not_found

### CODE REVIEW

File: packages/ketcher-react/src/icons/index.tsx [hunk 1]
```diff
@@ -215,7 +216,8 @@ const icons = {
   'shape-ellipse': ShapeEllipseIcon,
   'shape-rectangle': ShapeRectangleIcon,
   'shape-polyline': ShapePolylineIcon,
-  'shape-line': ShapeLineIcon
+  'shape-line': ShapeLineIcon,
+  'not-found': NotFound
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/icons/index.tsx:220]: Please follow naming convention

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ const RenderTmpl: FC<{
 const TemplateTable: FC<TemplateTableProps> = props => {
   const { templates, selected, onSelect, onDelete, onAttach } = props
   const ITEMS_COUNT = templates ? templates.length : 0
-  const ITEM_SIZE = { width: 178, height: 120 }
+  const ITEM_SIZE = { width: 100, height: 100 }
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx:67]: Could you move it into less file?

**Review by AndreiMazol — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/icons/index.tsx (hunk 1 above)
Comment (AnaMoskvina) [reply packages/ketcher-react/src/icons/index.tsx]: renamed

File: packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx (hunk 2 above)
Comment (AnaMoskvina) [reply packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx]: done

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**